packages/popup/src/pages/ImportPrivateKey/index.js

```diff
@@ -0,0 +1,137 @@
+import {useState, useEffect} from 'react'
+import {useHistory} from 'react-router-dom'
+import {useTranslation} from 'react-i18next'
+import {TextNav} from '../../components/index'
+import Button from '@cfxjs/component-button'
+import Input from '@cfxjs/component-input'
+import {CompWithLabel} from '../../components'
```

Comment:
combine this and TextNav together
vscode has the organize imports code action built in https://code.visualstudio.com/docs/languages/typescript#_organize-imports

Reply:
done